### PR TITLE
Fix magento2 strip config

### LINKED ISSUE #270: [magento2] strip-magento-config removes minification settings
The following should persist in `config.php` after `strip-magento-config` runs during the install process for magento2:
```
    'system' => [
        'default' => [
            'general' => [
                'locale' => [
                    'code' => 'en_GB'
                ]
            ],
            'dev' => [
                'js' => [
                    'minify_files' => '1',
                    'minify_exclude' => '
                    /tiny_mce/
                '
                ],
                'css' => [
                    'minify_files' => '1',
                    'minify_exclude' => '
                    /tiny_mce/
                '
                ],
                'static' => [
                    'sign' => '1'
                ]
            ]
        ]
    ]
```

### PR DESCRIPTION
**Pull request by kierenevans**
Fixes https://github.com/inviqa/harness-base-php/issues/270

### CODE REVIEW

**Comment by andytson-inviqa** [comment]
This seems to break the build atm on the magento2 steps

**Comment by kierenevans** [comment]
The 500 response during behat was down to php-fpm's docker container having the original config.php, and console's docker container having the config.php that was the result of `app:config:dump`.

Magento running via php-fpm detected a difference and refused to serve the application.

Usually project setups are done in dynamic mode where config.php will be the same in all containers via a mountpoint, so this only affects our "from nothing" static tests in Jenkins.

**Comment by andytson-inviqa** [comment]
not project CI or pipeline environments though, which are effectively from nothing

File: src/magento2/application/skeleton/app/etc/config.php [hunk 1]
```diff
@@ -504,6 +504,38 @@
...
+                ],
+                'static' => [
+                    'sign' => '1'
+                ],
+                'template' => [
+                    'minify_html' => '0'
+                ]
+            ]
+        ]
+    ],
```
Comment (andytson-inviqa) [inline src/magento2/application/skeleton/app/etc/config.php:538]: isn't this then fragile in that any environment that has different configuration will fail?

**Comment by kierenevans** [comment]
I guess this could happen in project CI environments if config.php is not representative of the extra settings.

**Comment by andytson-inviqa** [comment]
and pipeline environments?

**Comment by kierenevans** [comment]
Only when starting an environment up without a database, triggering the installer.

**Review by andytson-inviqa — APPROVED** [review]

**Pull request merged**